urban-os/values.yaml

```diff
@@ -79,3 +88,282 @@ kubernetes-data-platform:
+              labels:
+                severity: warning
+              annotations:
+                description: "Kubernetes cluster has {{ $value }} cores left. New deployments and cron jobs may fail to launch."
+                summary: "Kubernetes cluster low on CPU cores"
+            - alert: LowClusterMemory
+              expr: (cluster:capacity_memory_bytes:sum - cluster:guarantees_memory_bytes:sum) < 1000000000 #1GB
+              labels:
+                severity: warning
+              annotations:
+                description: "Kubernetes cluster has {{ $value | humanize }} memory left. New deployments and cron jobs may fail to launch."
+                summary: "Kubernetes cluster has less than {{ 1000000000.0 | humanize }} memory available"
+            - alert: NotReady
+              annotations:
+                description: "{{ $labels.node }} is not ready"
+                summary: "Status not Ready on Node {{ $labels.node }}"
+              expr: kube_node_status_condition{condition="Ready", status="true"} != 1
+              labels:
+                severity: error
+        - name: tracer_alerts
+          rules:
+            - alert: MessageThroughput
+              expr: sum(rate(kafka_topic_partition_current_offset{topic=~"raw-00000000-7e77-4b1c-92a4-36e09db56173|transformed-00000000-7e77-4b1c-92a4-36e09db56173|streaming-persisted|scos__sample_streaming_dataset"}[5m])) by (topic) == 0
+              labels:
+                severity: error
+              annotations:
+                summary: "{{ $labels.topic }} has no input for at least 5 minutes"
+        - name: joomla_backup
+          rules:
+            - alert: JoomlaBackupNotCompleted
+              expr: >
+                time() - max(kube_job_status_start_time{job_name=~"joomla-backup.*"}) > 1800 and
+                max(kube_job_status_completion_time{job_name=~"joomla-backup.*"}) < max(kube_job_status_start_time{job_name=~"joomla-backup.*"})
+              labels:
+                severity: error
+              annotations:
+                summary: "Joomla Backup Not Completed"
+                description: "Most recent Joomla backup did not complete"
+            - alert: JoomlaBackupOverdue
+              expr: >
+                time() - max(kube_job_status_completion_time{job_name=~"joomla-backup.*"}) > 172800
+              labels:
+                severity: error
+              annotations:
+                summary: "Joomla Backup Overdue"
+                description: "Joomla backup has not run in over 24 hours"
+        - name: kube_bench_failed
+          rules:
+            - alert: KubeBenchFailed
+              expr: kube_job_status_failed{job_name=~"kube-bench.*"}[25h]
+              labels:
+                severity: warning
+              annotations:
+                summary: "Kube Bench failed some checks"
+                description: "The most recent Kube Bench run detected some failures"
```

Comment:
Kube bench runs are not part of the standard urbanos deployment. This can be removed altogether.

Reply:
removed